### PR TITLE
CLEAR: typo fix for CSRF

### PR DESCRIPTION
**Pull request by jonathanKingston**

### CODE REVIEW

File: specs/clear-site-data/index.html [hunk 1]
```diff
@@ -1194,7 +1194,7 @@ <h4 class="heading settled" data-level="3.4.3" id="clear-cache"><span class="sec
   Chrome clears out prerendered pages, script caches, WebGL shader caches,
   WebRTC bits and pieces, address bar suggestion caches, various networking
   bits that aren’t representations (HSTS/HPKP, SCDH, etc.). Perhaps
-  <a data-link-type="biblio" href="#biblio-storage">[STORAGE]</a> will make this clearer?</p>
+  <span>[STORAGE]</span> will make this clearer?</p>
```
Comment (mikewest) [inline specs/clear-site-data/index.html]: Hrm. Do you have an outdated version of Bikeshed locally? Mind updating and trying again?

File: specs/clear-site-data/index.src.html [hunk 2]
```diff
@@ -198,7 +198,7 @@ <h4 id="example-signout">Signing Out</h4>
 
   <h4 id="example-targeted">Targeted Clearing</h4>
 
-  A user signs out of Megacorp Inc.'s site via a CSRD-protected POST to
+  A user signs out of Megacorp Inc.'s site via a CSRF-protected POST to
```
Comment (mikewest) [inline specs/clear-site-data/index.src.html]: *faceplam* Yes, thank you. :)

File: specs/clear-site-data/index.html (hunk 1 above)
Comment (jonathanKingston) [inline specs/clear-site-data/index.html]: That wasn't the issue it seems I ran it in the wrong dir so it couldn't find the biblio:

```
bikeshed -f spec index.src.html --md-Repository=w3c/webappsec
```

This seems to fix it:
```
bikeshed -f spec clear-site-data/index.src.html --md-Repository=w3c/webappsec
```

**Comment by mikewest** [comment]
Next time, you can just run `make CLEAR` in the specs/ directory. Should be aliases for all the Bikeshed specs there.

Thanks!

**Comment by jonathanKingston** [comment]
@mikewest yup missed that `make` too; equal facepalm. Thanks

**Pull request merged**